src/components/dev-grid-utils.js

```diff
@@ -1,58 +1,58 @@
 export const profiles = [
   {
-    githubUrl: 'https://github.com/MightyJoeW',
-    githubUsername: 'mightyjoew',
-    linkedinUrl: 'https://www.linkedin.com/in/josephmwarren/',
-    photo: 'https://avatars.githubusercontent.com/u/24509848?v=4',
-    siteUrl: 'https://www.getscriptordietryin.com/',
+    githubUrl: "https://github.com/MightyJoeW",
```

Comment:
It looks like we have different style rules for single vs. double quotes, but it's not a show stopper (it may get switched back to single quotes in the future).